graphql-cli/src/test/resources/expectedGenCode/client/apiKeysConfig/client.bal

```diff
@@ -6,21 +6,33 @@ public isolated client class GraphqlClient {
     final readonly & ApiKeysConfig apiKeysConfig;
     public isolated function init(ApiKeysConfig apiKeysConfig, string serviceUrl, ConnectionConfig config) returns graphql:ClientError? {
         http:ClientConfiguration httpClientConfig = {
-            httpVersion: config.httpVersion,
-            http1Settings: {...config.http1Settings},
-            http2Settings: config.http2Settings,
-            timeout: config.timeout,
-            forwarded: config.forwarded,
-            poolConfig: config.poolConfig,
-            cache: config.cache,
-            compression: config.compression,
-            circuitBreaker: config.circuitBreaker,
-            retryConfig: config.retryConfig,
-            responseLimits: config.responseLimits,
-            secureSocket: config.secureSocket,
-            proxy: config.proxy,
-            validation: config.validation
+           httpVersion: config.httpVersion,
+           timeout: config.timeout,
+           forwarded: config.forwarded,
+           poolConfig: config.poolConfig,
+           compression: config.compression,
+           circuitBreaker: config.circuitBreaker,
+           retryConfig: config.retryConfig
```

Comment:
Recheck if the record fields are aligned properly. It shows a small difference from the original. Consider the fields below too.